dashboard/config/scripts/levels/New PoemBot Project.level

```diff
@@ -0,0 +1,103 @@
s\",\"toolbox_blocks\",\"block_pools\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-04-30 12:01:17 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"helper_libraries\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-04-30 12:08:23 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"include_shared_functions\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-04-30 13:01:45 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"hide_animation_mode\",\"use_default_sprites\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-04-30 13:04:56 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-04-30 14:27:44 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-05-04 15:21:26 -0700\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 08:44:45 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"helper_libraries\",\"block_pools\",\"preload_asset_list\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 08:49:23 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"block_pools\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 08:56:39 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 09:24:09 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 10:35:35 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 10:43:51 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"},{\"changed_at\":\"2021-06-21 13:28:41 -0400\",\"changed\":[\"start_blocks\",\"toolbox_blocks\",\"preload_asset_list\",\"encrypted_examples\"],\"changed_by_id\":1,\"changed_by_email\":\"anjali+labs@code.org\"}]",
```

Comment:
Some of this history looks unrelated :)

Reply:
I think this is related to the bug where you can't actually make a new level on levelbuilder. So every new level has to be cloned from something else, and I think a lot of this history carries over when you clone.